src/ts/components/playbacktimelabel.ts

```diff
@@ -132,7 +134,15 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
       }
     };
 
+    let updateTimeFormatBasedOnDuration = () => {
+      // Set time format depending on source duration
+      this.timeFormat = Math.abs(player.isLive() ? player.getMaxTimeShift() : player.getDuration()) >= 3600 ?
+      StringUtils.FORMAT_HHMMSS : StringUtils.FORMAT_MMSS;
```

Comment:
Little formating for better readability:

```suggestion
      this.timeFormat = Math.abs(player.isLive() ? player.getMaxTimeShift() : player.getDuration()) >= 3600 ?
        StringUtils.FORMAT_HHMMSS : StringUtils.FORMAT_MMSS;
```